WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/ApllicationLogDetailView.swift

```diff
@@ -0,0 +1,103 @@
+import SwiftUI
+
+/// Hosting controller that wraps an `ApplicationLogDetailView`
+///
+final class ApplicationLogDetailViewController: UIHostingController<ApplicationLogDetailView> {
+    init(with contents: String, for date: String) {
+        let viewModel = ApplicationLogViewModel(logText: contents)
```

Comment:
I've seen some places in the app where is preferred to pass the already created `ViewModel`
I think we should follow a common standard but there is none at the moment? What do you think?

Reply:
I don't have a preference, but if that's what we're doing elsewhere, it makes more sense to do it that way. Added it in 8585d9643